src/qtbase/QColor.js

```diff
@@ -1,32 +1,78 @@
-// TODO complete implementation (with attributes `r`,`g` and `b`).
-
 class QColor {
   constructor(val) {
+    this.r = this.g = this.b = 0;
```

Comment:
Why no save `r g b` as 0~255 replace 0~1 in constructor. 
For readability and could be faster than `Math.round(v * 255);` in function `$css` and `toString`.